base/notification/notification.go

```diff
@@ -57,6 +56,8 @@ type Notification struct {
 	// Recipients settings - Applications can add extra email recipients by adding entries to this array.
 	// This setting extends whatever the Administrators configured in their Notifications settings (since v1.1.0).
 	Recipients []Recipient `json:"recipients,omitempty"`
+	// org_id as future replacement of the account_id.
+	OrgID string `json:"org_id,omitempty"`
```

Comment:
why can't we use `OrgID`? If notifications support it, you can use it right away, we should be using OrgID everywhere where it is possible.

Reply:
Notifications REST API accepted integer only and for string raised `OrgID must be greater than 0` error, while API spec clearly said it wants string. Today REST API returns 302 moved permanently, so they are doing something with it. I am fine with passing it but my priority is to test notifications now and it can cause some delay if we would need to revert it.

Reply:
I would pass it after testing to be safe :slightly_smiling_face:

Reply:
sure, thank you for explanation